### ISSUE TITLE
Sync definitions from main web site/PDF to definitions section in Elements docs

### ISSUE THREAD

**Issue by davewilks** [cloud-elements/devportal#435]
## Description
See the attachment for definitions.
[cloud-elements-api-hubs-product-definitions.pdf](https://drive.google.com/a/cloud-elements.com/file/d/0BwWlI1jhc4d0MjdITi1WeUViS0U/view?usp=sharing)

**Comment by daniel-cherry** [comment]
@davewilks The attachment link above does not appear to open a PDF file. Is this product tools or enablement?

**Comment by davewilks** [comment]
[Uploading cloud-elements-api-hubs-product-definitions.pdf…]()

**Comment by davewilks** [comment]
I updated the link.

**Comment by daniel-cherry** [comment]
@davewilks This looks related to #374

**Comment by davewilks** [comment]
I agree. We can close one as a duplicate, probably the Glossary one.

**Comment by davewilks** [comment]
https://docs.google.com/document/d/11sC1zP3qeocQVmGBe7WCP73BO7c-krGMhMLpJMypkwg/edit?ts=58b45dcf
Google Docs version of the PDF
